test/integration/default/serverspec/check-load_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'shared_spec'
+
+gem_path = '/usr/local/bin'
+check_name = 'check-load.rb'
+check = "#{gem_path}/#{check_name}"
+
+describe 'ruby environment' do
+  it_behaves_like 'ruby checks', check
+end
+
+describe file(check) do
```

Comment:
probably best to put all the common checks into something like a spec helper that can be called, here is an example: https://github.com/sensu-plugins/sensu-plugins-dcos/blob/0.1.1/test/integration/helpers/serverspec/shared_spec.rb

https://github.com/sensu-plugins/sensu-plugins-dcos/blob/0.1.1/test/integration/default/serverspec/check-dcos-component-health.rb_spec.rb#L10-L12